src/libslic3r/GCode.cpp

```diff
@@ -3741,6 +3741,8 @@ std::string GCode::_before_extrude(const ExtrusionPath &path, const std::string
             speed = m_config.get_abs_value("ironing_speed");
         } else if (path.role() == erNone) {
             speed = m_config.get_abs_value("travel_speed");
+        } else if (path.role() == erNone) {
+            speed = m_config.get_abs_value("travel_speed_z");
```

Comment:
shouldn't you also test the path to check taht it contains only z-move? and put the if before the travel_spedd one or it will never tigger

Reply:
Actually this is not required at all, since the speed is set in `GCodeWriter::_travel_to_z`.